image-swipe.ios.ts

```diff
@@ -135,6 +136,26 @@ export class ImageSwipe extends ImageSwipeBase {
         imageView.frame = contentsFrame;
     }
 
+    public nextPage(): void {
+        this._animateLoadPage = true;
+        super.nextPage();
+    }
+
+    public prevPage(): void {
+        this._animateLoadPage = true;
+        super.prevPage();
+    }
+
+    private get _animateLoadPage(): boolean {
+        const current: boolean = this._animateLoadPageValue;
+        this._animateLoadPageValue = false;
```

Comment:
This is a really "hidden" thing and it is not very intuitive that the flag will be lowered once it is returned. How about we change this to a simple property and just raise the flag before navigation and the lower it manually?